corehq/apps/reports/management/commands/project_stats_report.py

```diff
@@ -339,8 +387,220 @@ def _get_table_size(config):
             for datasource in static_datasources + dynamic_datasources
         ], key=lambda r: r[-1] if r[-1] != 'Table not found' else 0)
 
+        resource_model['datasources_info'] = rows
+
+    def collect_case_transactions(self):
+        db_name = get_db_aliases_for_partitioned_query()[0]
+        db_cursor = connections[db_name].cursor()
+
+        with db_cursor as cursor:
+            cursor.execute("""
+                SELECT COUNT(*) as num_forms, d.count as num_updates
+                FROM (
+                    SELECT COUNT(*) as count
+                    FROM form_processor_casetransaction
+                    WHERE case_id IN (
+                        SELECT case_id
+                        FROM form_processor_commcarecasesql
+                        WHERE domain = %s
+                    )
+                    GROUP BY form_id
+                ) AS d
+                GROUP BY d.count;
+            """, [self.domain])
+            result = cursor.fetchall()
+
+            running_form_case_updates = 0
+            total_forms = 0
+            for num_forms, num_updates in result:
+                total_forms += num_forms
+                running_form_case_updates += num_forms * num_updates
+
+            resource_model['case_transactions_per_form'] = running_form_case_updates / total_forms
+
+            cursor.execute("""
+                SELECT COUNT(*)
+                FROM form_processor_casetransaction
+                WHERE case_id IN (
+                    SELECT case_id
+                    FROM form_processor_commcarecasesql
+                    WHERE domain = %s
+                );
+            """, [self.domain])
+            (total_transactions,) = cursor.fetchone()
+            resource_model['case_transactions_total'] = total_transactions
+
+    def collect_case_indices(self):
+        db_name = get_db_aliases_for_partitioned_query()[0]
+        db_cursor = connections[db_name].cursor()
+
+        with db_cursor as cursor:
+            cursor.execute("""
+                SELECT COUNT(*)
+                FROM form_processor_commcarecaseindexsql
+                WHERE domain = %s;
+            """, [self.domain])
+            (total_case_indices,) = cursor.fetchone()
+
+            total_cases = resource_model['cases_total']
+            if total_cases > 0:
+                resource_model['case_indices'] = total_case_indices/total_cases
+
+    def collect_synclogs(self):
+        db_name = router.db_for_read(SyncLogSQL)
+        db_cursor = connections[db_name].cursor()
+
+        with db_cursor as cursor:
+            cursor.execute("""
+                SELECT COUNT(*), d.count
+                FROM (
+                    SELECT COUNT(*) AS count
+                    FROM phone_synclogsql
+                    WHERE domain = %s
+                    GROUP BY user_id
+                ) AS d
+                GROUP BY d.count
```

Comment:
similar comment to above about using `sum` instead of doing this weird group by count.